marklogic-data-hub/src/main/java/com/marklogic/hub/impl/DataHubImpl.java

```diff
@@ -530,15 +527,10 @@ public void install(HubDeployStatusListener listener) {
                 throw new DataHubConfigurationException(e);
             }
         }
-        AppConfig appConfig = hubConfig.getAppConfig();
```

Comment:
3.15.0 defaults to using CMA for these resources, so no longer need to set these.